src/librustc_typeck/lib.rs

```diff
@@ -379,7 +379,7 @@ pub fn check_crate<'a, 'tcx>(tcx: TyCtxt<'a, 'tcx, 'tcx>)
     }
 }
 
-/// A quasi-deprecated helper used in rustdoc and save-analysis to get
+/// A quasi-deprecated helper used in rustdoc and clippy to get
```

Comment:
cc @oli-obk @Manishearth We should maybe have an issue for getting clippy off of this.

Reply:
cc @flip1995 in case this is on your list of clippy utilities